app.py

```diff
@@ -35,6 +24,60 @@
     app.config["SC_API"] = None
 
 
+# Register actions.
+# 
+# Actions are functions with the signature `(api_auth, parameters, contexts) -> response_string`
+actions = {}
+
+def register_action(name, func):
+    actions[name] = func
+
+from actions.create_site import create_site
```

Comment:
Would prefer import statements at the top of the file per PEP8: https://www.python.org/dev/peps/pep-0008/#imports

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Reply:
That makes sense, but in this case I argue it is clearer to put them next to the one place they are used. These aren't wider-use imports like libraries, they're specifically importing the action function that is only used to register the action.

If you still stand by your comment I'll change it, but that's the reason I did it.

Reply:
Please keep it a standard to put all imports at the top